quipucords/api/common/common_report.py

```diff
@@ -55,7 +55,7 @@ def extract_tar_gz(file_like_obj):
         tar_name = f"/tmp/api_tmp_{time.strftime('%Y%m%d_%H%M%S')}.tar.gz"
         with open(tar_name, "wb") as out_file:
             out_file.write(file_like_obj)
-        tar = tarfile.open(tar_name)  # pylint: disable=consider-using-with
+        tar = tarfile.open(tar_name)
```

Comment:
I figure we don't have time to fix everything that this PR touches, but _yikes_… 😬

I will 🙈 and pretend I didn't see this (and probably other disabled violations that should never have been disabled).